apps/cow-tools/src/app/milkman/utils/readTokenDecimals.ts

```diff
@@ -0,0 +1,20 @@
+import { erc20ABI } from "@wagmi/core";
+import { Address, createPublicClient, http } from "viem";
+import { goerli } from "viem/chains";
+
+export const publicClient = createPublicClient({
+  chain: goerli,
+  transport: http(),
+});
+
+export async function readTokenDecimals(tokenAddress: Address) {
```

Comment:
do we even need this? Couldn't we just read from CoW's token list? https://github.com/cowprotocol/token-lists
[files.cow.fi/tokens/CowSwap.json](https://files.cow.fi/tokens/CowSwap.json)